Header header
string name
Waypoint[] waypoints
bool cyclic
